README.md

```diff
@@ -162,22 +165,23 @@ curl \
+| ---------------------------------- | ----------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `GET /health`                      | `200`             | `OK`                                                                                                                    |
+|                                    | `403`             | `Forbidden`                                                                                                             |
+| `GET /readiness`                   | `200`             | `OK`                                                                                                                    |
+|                                    | `403`             | `Forbidden`                                                                                                             |
+|                                    | `502`             | `Bad Gateway` when can not connect to Bee node                                                                          |
+| `GET /bzz/:swarmhash`              | `200`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/BZZ/paths/~1bzz~1{reference}/get)                   |
+| `POST /bzz`                        | `201`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/BZZ/paths/~1bzz/post)                               |
+| `GET /bytes/:swarmhash`            | `200`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Bytes/paths/~1bytes~1{reference}/get)               |
+| `POST /bytes`                      | `201`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Bytes/paths/~1bytes/post)                           |
+| `GET /chunks/:swarmhash`           | `200`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Chunk/paths/~1chunks~1{reference}/get)              |
+| `POST /chunks`                     | `201`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Chunk/paths/~1chunks/post)                          |
+| `POST /soc/:owner/:id`             | `201`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Single-owner-chunk/paths/~1soc~1{owner}~1{id}/post) |
+| `GET /feeds/:owner/:topic`         | `200`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Feed/paths/~1feeds~1{owner}~1{topic}/get)           |
+| `POST /feeds/:owner/:topic`        | `201`, `403`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Feed/paths/~1feeds~1{owner}~1{topic}/post)          |
+| `PATCH /stamps/topup/:id/:amount`  | `202`, `400`, ... | See official [bee documentation](https://docs.ethswarm.org/api/#tag/Postage-Stamps/paths/~1stamps~1topup~1{id}~1{amount}/patch)          |
```

Comment:
```suggestion
```

This is not true. This documents the API of the gateway-proxy and not API that the gateway-proxy internally uses ;-) And to my knowledge, you don't directly exposes this API to the gateway-proxy's user, so please remove it.

Reply:
@AuHau in this section, what are the parameters to know what endpoints we put according to the API methods we use?

Reply:
Well this section is describing the API of the gateway-proxy ;-) Not the Bee API...

Reply:
Sorry, now I get why it is so misleading. Gateway Proxy is proxying some of the endpoints from Bee and hides the unnecessary & potentially vulnerable ones. You can see the list here: https://github.com/ethersphere/gateway-proxy/blob/master/src/server.ts#L12

So this table shows which endpoints are available (also those proxying from Bee). But the Stamps API is not exposed ;-) It is only used by gateway-proxy internally.